include/nccl_ofi_rdma.h

```diff
@@ -343,6 +343,12 @@ typedef struct nccl_net_ofi_rdma_send_comm {
 	/* Comm ID provided by remote endpoint */
 	uint64_t remote_comm_id;
 
+	/* Request to send connect message */
```

Comment:
this isn't a request to send a connect message.  It's the request tracking the connect message.